cli/internal/fs/turbo_json.go

```diff
@@ -126,7 +126,7 @@ func LoadTurboConfig(rootPath turbopath.AbsoluteSystemPath, rootPackageJSON *Pac
 	if !includeSynthesizedFromRootPackageJSON && err != nil {
 		// If the file didn't exist, throw a custom error here instead of propagating
 		if errors.Is(err, os.ErrNotExist) {
-			return nil, fmt.Errorf("Could not find %s. Follow directions at https://turbo.build/repo/docs to create one: file does not exist", configFile)
+			return nil, errors.Wrapf(err, "Could not find %s. Follow directions at https://turbo.build/repo/docs to create one", configFile)
```

Comment:
Might want to rebase on `main` first. I think I did this in #2706 (smaller diff in #3760)
https://github.com/vercel/turbo/blob/5b44c9289e7d72f05745342ca7a1c6f63479bf0a/cli/internal/fs/turbo_json.go#L171